### ISSUE TITLE
Phishtank is returning XML instead of JSON

### ISSUE THREAD

**Issue by astaruch** [scone/pyphishtank#1]
Hi,

this is probably dead project, but I am playing with a PhishTank API and I have a problem, that responses are coming only in XML, even when `format='json'` is in request parameters. Then I've found your project and saw, that you are parsing JSON a returning a response.

Does this work for you? When I downloaded a debugged your code, the response is obtained in XML despite the fact that you want a JSON.

I am suspecting that they have changed the API/disabled JSON format, but I'd like to know if it's been working in the past.

Thanks!

**Comment by pavankomarina** [comment]
@astaruch Yes, you are correct. It's returning XML right now. Did you find any other way to get the JSON response?  If yes please share it here
